### PR TITLE
Message when the video doesn't exist in playlist

### PR DESCRIPTION
**Pull request by unixfox**
Fixes https://github.com/iv-org/invidious/issues/3242

### CODE REVIEW

**Comment by SamantazFox** [comment]
Shouldn't we just ignore the `list` and `index` parameters if nothing was found?

**Comment by unixfox** [comment]
> Shouldn't we just ignore the `list` and `index` parameters if nothing was found?
> 
> Edit: Nevermind, I didn't see that the embed was embedding a playlist directly, not a video....

Good to merge?

**Comment by unixfox** [comment]
> > Shouldn't we just ignore the `list` and `index` parameters if nothing was found?
> > Edit: Nevermind, I didn't see that the embed was embedding a playlist directly, not a video....
> 
> Good to merge?

^
@SamantazFox ?

**Comment by SamantazFox** [comment]
We should translate the message! (See #1497)

**Comment by unixfox** [comment]
> We should translate the message! (See #1497)

How do you even do that with HTML code returned?

**Comment by unixfox** [comment]
Added translation, I hope I did it the right way.

File: locales/en-US.json [hunk 1]
```diff
@@ -471,5 +471,6 @@
     "crash_page_switch_instance": "tried to <a href=\"`x`\">use another instance</a>",
     "crash_page_read_the_faq": "read the <a href=\"`x`\">Frequently Asked Questions (FAQ)</a>",
     "crash_page_search_issue": "searched for <a href=\"`x`\">existing issues on GitHub</a>",
-    "crash_page_report_issue": "If none of the above helped, please <a href=\"`x`\">open a new issue on GitHub</a> (preferably in English) and include the following text in your message (do NOT translate that text):"
+    "crash_page_report_issue": "If none of the above helped, please <a href=\"`x`\">open a new issue on GitHub</a> (preferably in English) and include the following text in your message (do NOT translate that text):",
+    "video_not_exist_in_playlist": "The video requested doesn't exist in the playlist. Click  <a href=\"`x`\">here</a> for the playlist home page."
```
Comment (SamantazFox) [inline locales/en-US.json:475]:
```suggestion
    "error_video_not_in_playlist": "The requested video doesn't exist in this playlist. <a href=\"`x`\">Click  here for the playlist home page.</a>"
```

Comment (SamantazFox) [reply locales/en-US.json:475]: Changes proposed:
* `error_` prefix in error messages' IDs 
* Minor wording changes
* Links containing non-descriptive text (like "click here") should be avoided, as they aren't accessible => using the whole phrase as the link's text.

**Review by SamantazFox — CHANGES_REQUESTED** [review]
Small correction on the translation side, otherwise looking good.

File: locales/en-US.json (hunk 1 above)
Comment (unixfox) [reply locales/en-US.json]: Done

**Pull request merged**